src/commands/dependabot/consolidate.ts

```diff
@@ -122,6 +122,9 @@ export default class Consolidate extends SfdxCommand {
           this.log(`${green('Created Pull Request:')} ${response.data._links.html.href}`);
         }
       } catch (err) {
+        if (!(err instanceof Error) || typeof err !== 'string') {
```

Comment:
there's a few of these.  Another approach would be to have the `error` method accept and handle a wider variety of things to the consumers don't have to.